src/main/kotlin/org/wycliffeassociates/otter/jvm/testapp/app/RecordableVMTestApp.kt

```diff
@@ -0,0 +1,9 @@
+package org.wycliffeassociates.otter.jvm.testapp.app
+
+import tornadofx.*
+
+class RecordableVMTestApp: App(RecordableVMTestView::class) {
```

Comment:
Did you mean to check in these test classes?  I'm fine with it if you are.

Reply:
Nope, I didn't mean to check in the testapp classes. I removed them in the latest commit.